common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4411,6 +4411,11 @@
           "<functionName> with AES-<mode> does not support initialization vectors (IVs)."
         ]
       },
+      "ALTER_NAMESPACE_PROPERTY" : {
```

Comment:
Rename `SET_NAMESPACE_PROPERTY` to `ALTER_NAMESPACE_PROPERTY`, We will reuse it in `SparkSqlParser`

Reply:
In SQL, SET also means update, I think it's fine

Reply:
Because in the following two cases, it is `UNSUPPORTED_FEATURE.SET_NAMESPACE_PROPERTY`, and I am concerned that it may cause misunderstandings. 

**test reserved properties**
```
sql(s"CREATE NAMESPACE $ns")
val sqlText = s"ALTER NAMESPACE $ns SET PROPERTIES ('$key'='dummyVal')"
```

```
sql(s"CREATE NAMESPACE $ns")
val sqlText = s"ALTER NAMESPACE $ns UNSET PROPERTIES ('$key')"
```

Okay, let me restore it

Reply:
The corresponding errors for table properties: `UNSUPPORTED_FEATURE.SET_TABLE_PROPERTY`, eg:
```
ALTER TABLE ... SET PROPERTIES ('$reservedKey'='...')"
ALTER TABLE ... UNSET PROPERTIES ('$reservedKey')"
```